src/odemis/driver/picoquant.py

```diff
@@ -2207,7 +2207,7 @@ def _generate(self):
 
         # Read data and make it a DataArray
         d = self.parent.GetCountRate(self._channel)
-        nd = numpy.array([d], dtype=numpy.int)
+        nd = numpy.array([d], dtype=int)
```

Comment:
I think we should actually specify the int. Need to check what the GetCountRate() C function returns, and use the same type. Most like numpy.uint32.

Reply:
This has been fixed in: https://github.com/delmic/odemis/pull/2467